test/behavior/tendrl/cluster.feature

```diff
@@ -1 +1,91 @@
-Feature: CLUSTER
\ No newline at end of file
+Feature: CLUSTER
+
+  Scenario: Import a cluster
+    Given User is on "clusters" page
+     When User should see the "import" button
+      And User clicks on "import" button
+     Then User sees the "import-clusters" page
+
+  Scenario: Unmanage a cluster
+    Given User is on "clusters" page
+     When User clicks on cluster dropdown
+      And User clicks on unmanage button
+      And User sees "Unmanage Cluster" dialogue box
+      And User clicks on "unmanage" button
+     Then User should see "unmanage cluster" "task page"
+      And User closes "task page"
+      And User sees cluster is unmanaged
+ 
+  Scenario: Naming an imported cluster
```

Comment:
This has multiple scenarios.

Normal import
import with name
import with enable profiling 
import with disable profiling 
import with unchanged profiling 
normal import with enable profiling 
Normal import with disable profiling 
Normal import with unchanged profiling 

I suggest we create a new file for import

Reply:
I agree